app/src/main/java/io/apicurio/registry/metrics/health/liveness/PersistenceExceptionLivenessCheck.java

```diff
@@ -39,18 +39,18 @@ public class PersistenceExceptionLivenessCheck extends AbstractErrorCounterHealt
      * to 60.
      * TODO report the absolute count as a metric?
      */
-    @ConfigProperty(name = "registry.metrics.PersistenceExceptionLivenessCheck.counterResetWindowDurationSec", defaultValue = "60")
+    @ConfigProperty(name = "apicurio.metrics.PersistenceExceptionLivenessCheck.counterResetWindowDurationSec", defaultValue = "60")
     @Info(category = "health", description = "Counter reset window duration of persistence liveness check", availableSince = "1.0.2.Final")
     Integer configCounterResetWindowDurationSec;
 
     /**
      * If set to a positive value, reset the liveness status after this time window passes without any further errors.
      */
-    @ConfigProperty(name = "registry.metrics.PersistenceExceptionLivenessCheck.statusResetWindowDurationSec", defaultValue = "300")
+    @ConfigProperty(name = "apicurio.metrics.PersistenceExceptionLivenessCheck.statusResetWindowDurationSec", defaultValue = "300")
     @Info(category = "health", description = "Status reset window duration of persistence liveness check", availableSince = "1.0.2.Final")
     Integer configStatusResetWindowDurationSec;
 
-    @ConfigProperty(name = "registry.metrics.PersistenceExceptionLivenessCheck.disableLogging", defaultValue = "false")
+    @ConfigProperty(name = "apicurio.metrics.PersistenceExceptionLivenessCheck.disableLogging", defaultValue = "false")
```

Comment:
Unify `apicurio.metrics.PersistenceExceptionLivenessCheck.disableLogging` with other enabled/disabled properties, e.g. `apicurio.metrics.PersistenceExceptionLivenessCheck.logging.disabled`. Maybe even `apicurio.metrics.PersistenceExceptionLivenessCheck.log.disabled` if we decide to avoid using present continuous form.